tools/should-skip-ci.js

```diff
@@ -0,0 +1,95 @@
+import { readFileSync } from "fs";
+
+const SKIP_CACTI = "skip-cacti-ci";
+const MaintainersFile = "MAINTAINERS.md";
+//regular expression to get the maintainers in MAINTAINERS.md
+const MAINTAINERS_REGEX =
```

Comment:
This regex is too big for us to maintain in long run (Please refer to how we have added multiple regex in pr-commit-parity.js script). As the next steps, we can break this up into several simpler regex, which can be maintained in a better way.
Summary: Instead of doing maintainersData.replace(ONE_BIG_LONG_REGEX), you can cascade simpler, shorter regex for better maintainability like string.replace(HEADER_REGEX).replace(HYPHEN_REGEX).replace(DELIMITER_REGEX)... and so on...
I will create another issue to fix that once this gets merged